### ISSUE TITLE
Display of new functions after an update

### ISSUE THREAD

**Issue by Tealk** [thunder-app/thunder#349]
**Is your feature request related to a problem? Please describe.**
It's not easy to keep track of all the great new features that are available in an app.

**Describe the solution you'd like**
My idea would be that after an update the new features are presented in a kind of popup, here are a few criteria / ideas:

- It should be easy to close, a button that is always visible
- In the popup there should be an option for permanent deactivation
- It should be activatable and deactivatable, in the settings
- It should not be overloaded with many bugfixes, only the best/most beautiful should be shown

**Comment by Poolitzer** [comment]
I would argue no bug fixes are needed in that pop up unless its a bug fix only release. Just put the big new features in it.

One could even link to the author out of gratitude to them. Maybe even add it to the PR template so authors can decide if they want to be linked to or not.

**Comment by Tealk** [comment]
> One could even link to the author out of gratitude to them.

I would not include links, this rather fits into the full changelog

**Comment by micahmo** [comment]
I was just looking through issues and found this one. Turns out I pretty much implemented it without even knowing it was a request! 😆 See #1208.

> * It should be easy to close, a button that is always visible
> * In the popup there should be an option for permanent deactivation
> * It should be activatable and deactivatable, in the settings
> * It should not be overloaded with many bugfixes, only the best/most beautiful should be shown

I did all of these except the last. The in-app changelog shows the same contents as the release page (so that we don't have to maintain separate messaging for the app). However, the changelog always begins with a nice, high-level summary, and it's up to you if you wish to scroll through all the details or not, so I think it's a good compromise.

@Tealk and @Poolitzer Are you ok with the current implementation?
